src/Model/Parseable.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpTui\Tui\Model;
+
+interface Parseable
+{
+    public static function parse(string $string): self;
```

Comment:
This interface is a bit anaemic (it's about Style but the naming implies anything or everything) and it's role is more of a Parser and not something that can be parsed? - maybe rename to StyleParser

Reply:
I'm out of the computer right now, but, yeah. To be honest, I don't even know if it makes sense to have this interface at all. Thinking about dropping it for now... Unless you want me to keep it or change it. Can you please review this later (where it is used etc) and give me some feedback about this?

Reply:
yeah I don't think it makes sense atm